src/avro2json.c

```diff
@@ -718,6 +739,17 @@ static int avro_value_to_csv(FILE *dest, const avro_value_t *value,
     const void *val;
     size_t size;
     CHECKED_EV(avro_value_get_fixed(value, &val, &size));
+
+    if(conf->ms_hadoop_logical_types && size == 16){
```

Comment:
This block is a copy & paste from the same thing in conversion to json. Maybe move to a helper function?

Reply:
+1